articles/quickstart/native/ios-swift/03-user-sessions.md

```diff
@@ -96,24 +97,21 @@ credentialsManager.credentials { error, credentials in
 } 
 ```
 
-If they have expired the credentials manager will automatically renew them for you using the refresh token.
+If the credentials have expired, the credentials manager will automatically renew them for you with the refresh token.
 
-## On Logout: Clear the Keychain
+## Clear the Keychain When the User Logs Out
 
-Whenever you need to log the user out, you should remove the credentials from the keychain:
+When you need to log the user out, remove their credentials from the keychain:
 
 ```swift
 // SessionManager.swift
 
 credentialsManager.clear()
 ```
 
-That's it. You've already dealt with the basic concepts of session handling in your app.
-
-## Fetch the User Profile
+## Get the User Profile
 
-The first step is to fetch the user profile. To do so, you need a valid `accessToken` first, which you 
-can find in the credentials object returned by the credentials manager.
+To get the user's profile, you need a valid access token. You can find the token in the `credentials` object returned by the credentials manager.
```

Comment:
access token

Reply:
Access token mentioned in the previous sentence. This way, we're avoiding repetition and still being clear on which token is needed.